2.0.0/CHANGELOG.md

```diff
@@ -1,10 +1,12 @@
 # Changelog
 
-Astronomer Certified 2.0.0-8.dev, TBC
+Astronomer Certified 2.0.0-8, 2021-07-13
 ----------------------------------------
 
 ## Bugfixes
 
+- Handle Istio containers with Kubernetes Executor Pod adoption (#1318) ([commit](https://github.com/astronomer/airflow/commit/3f309b057))
```

Comment:
```suggestion
- [astro] Handle Istio containers with Kubernetes Executor Pod adoption (#1318) ([commit](https://github.com/astronomer/airflow/commit/3f309b057))
```

Some changelogs have the `[astro]` prefix, others don't?

Reply:
oh yes this one should, as it is astro specific commit